packages/app-aco/src/components/Tree/List/utils.ts

```diff
@@ -7,33 +7,37 @@ import { ROOT_ID } from "./constants";
  *
  * @param folders list of folders returned by useFolders hook.
  * @param focusedNodeId id of the current folder selected/focused.
+ * @param hiddenFolderId id of the folder you don't want to show within the list.
  * @return array of elements to render the tree component.
  */
 export const createTreeData = (
     folders: FolderItem[] = [],
-    focusedNodeId?: string
+    focusedNodeId?: string,
+    hiddenFolderId?: string
```

Comment:
We don't need this right now, since hiding one folder does not show the whole hierarchy under it. It would be like changing a few lines of code, giving us more flexibility. I'm working on it